public/pages/SampleData/utils/constants.tsx

```diff
@@ -183,19 +183,27 @@ export const sampleEcommerce = {
         },
       },
       {
-        feature_name: 'sum_total_revenue',
+        feature_name: 'avg_total_revenue',
         feature_enabled: true,
         importance: 2,
         aggregationQuery: {
-          sum_total_revenue: { sum: { field: 'total_revenue_usd' } },
+          avg_total_revenue: { avg: { field: 'total_revenue_usd' } },
         },
       },
       {
-        feature_name: 'avg_total_revenue',
+        feature_name: 'max_total_revenue',
         feature_enabled: true,
         importance: 3,
         aggregationQuery: {
-          avg_total_revenue: { avg: { field: 'total_revenue_usd' } },
+          max_total_revenue: { max: { field: 'total_revenue_usd' } },
+        },
+      },
+      {
+        feature_name: 'min_total_revenue',
+        feature_enabled: true,
+        importance: 4,
```

Comment:
The importance was removed. Why add importance in the sample detector?

Reply:
Maybe I'm confused about what importance is here, I was thinking it was related to the order of features shown in the UI, but I see that it's not under uiMetadata, so maybe not. Could you explain what it means?

Reply:
Original design was adding weight for feature. Feature with high importance will have more weight than others and will impact more for the anomaly result. But this design was removed later, no feature "importance" now.

Reply:
I see, thanks for clarifying. Will remove importance for all of the features.